### ISSUE TITLE
tsc-wrapped throws "File not found" error on Windows

### ISSUE THREAD

**Issue by jelbourn** [angular/angular#15403]
Reproduction: https://github.com/jelbourn/material2/tree/windows-error-repro

Steps to reproduce:
1. `npm i`
2. `gulp library:build:esm`

Error
```
[14:43:25] Error: Error File 'C:/code/material2/src/lib/index.ts' not found.
    at UserError.Error (native)
    at new UserError (C:\code\tools\@angular\tsc-wrapped\src\tsc.ts:33:5)
    at Object.check (C:\code\tools\@angular\tsc-wrapped\src\tsc.ts:81:11)
    at Object.main (C:\code\tools\@angular\tsc-wrapped\src\main.ts:109:5)
    at Gulp.<anonymous> (C:\code\material2\tools\gulp\tasks\library.ts:44:33)
    at module.exports (C:\code\material2\node_modules\orchestrator\lib\runTask.js:34:7)
    at Gulp.Orchestrator._runTask (C:\code\material2\node_modules\orchestrator\index.js:273:3)
    at Gulp.Orchestrator._runStep (C:\code\material2\node_modules\orchestrator\index.js:214:10)
    at Gulp.Orchestrator.start (C:\code\material2\node_modules\orchestrator\index.js:134:8)
    at C:\Users\Angular\AppData\Roaming\npm\node_modules\gulp\bin\gulp.js:129:20
```

This same build set up works on Linux and OSX. 

Context:
This is the build setup for Angular Material, which calls the `main` method of `tsc_wrapped` directly (passing the tsconfig path).

**Comment by nehaagrawal** [comment]
any updates on this issue? I am getting similar error.

**Comment by chuckjaz** [comment]
This was fixed and no longer reproduces in `4.0.2`.

**Comment by ashushre** [comment]
Error: Cannot find module '@angular/tsc-wrapped/src/tsc' how to solve in ionic2
I m having this type of error
